libs/utils/utils.js

```diff
@@ -737,6 +763,13 @@ export function decorateLinks(el) {
         processQuickLink(a);
       })();
     }
+
+    if (a.href.includes(BUSINESS_PLANS_PATH)) {
```

Comment:
Have you explored using an autoblock for such scenario? I think it would be better instead of adding all the logic in utils.

Reply:
@bozojovicic - you can see `caas` or `faas` autoblocks for reference

Reply:
I created it `libs/blocks/m7business/m7business.js`
BTW, autoblock is awesome idea!

Reply:
Thank you @bozojovicic ! I have cleared my request for changes, things look good now. Thanks!